snapcraft/internal/lifecycle.py

```diff
@@ -254,15 +256,35 @@ def snap(project_options, directory=None, output=None):
 
     snap_name = output or common.format_snap_name(snap)
 
-    logger.info('Snapping {}'.format(snap_name))
     # These options need to match the review tools:
     # http://bazaar.launchpad.net/~click-reviewers/click-reviewers-tools/trunk/view/head:/clickreviews/common.py#L38
     mksquashfs_args = ['-noappend', '-comp', 'xz', '-no-xattrs']
     if snap['type'] != 'os':
         mksquashfs_args.append('-all-root')
 
-    subprocess.check_call(
-        ['mksquashfs', snap_dir, snap_name] + mksquashfs_args)
+    ticker = '/-\\|'
+    i = 0
+    with subprocess.Popen(
+            ['mksquashfs', snap_dir, snap_name] + mksquashfs_args,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT) as proc:
+        ret = None
+        if os.isatty(sys.stdout.fileno()):
+            ret = proc.poll()
+            while ret is None:
+                print('\033[0;32m\rSnapping {!r}\033[0;32m {}'.format(
+                    snap['name'], ticker[i]), end='')
+                i = (i+1) % len(ticker)
+                time.sleep(.2)
+                ret = proc.poll()
```

Comment:
In other parts of the code (I'm thinking of the upload specifically) we used `progressbar` for this. Should we maintain consistency?